src/MooLite/core/server/messages/InitCharacterInfoParser.ts

```diff
@@ -39,7 +39,7 @@ export class InitCharacterInfoParser extends MessageParser {
         game.abilities.updateCombatTriggers(message.abilityCombatTriggersMap);
         game.actionQueue.updateActions(message.characterActions);
         game.inventory.updateCharacterItems(message.characterItems, false);
-        game.inventory.updateCharacterDrink(message.actionTypeDrinkSlotsMap);
+        game.inventory.updateCharacterDrinks(message.actionTypeDrinkSlotsMap);
         game.inventory.updateCharacterFood(message.actionTypeFoodSlotsMap);
```

Comment:
This update and the drink one can be called with `notify = false` to avoid triggering events on the initial load!

Reply:
Ah, I see. Thanks for pointing that out!